### PR TITLE
[FR] French Translation game modes

### PR DESCRIPTION
**Pull request by Gastonia02**
main_fr-5_page-0001

### CODE REVIEW

File: translations/game_modes.tex/fr.po [hunk 1]
```diff
@@ -91,6 +97,11 @@ msgid ""
 "Further rules changes are detailed in the Campaign Mission Books.\n"
 "\n"
 msgstr ""
+"\\subsection*{Campagne}\n"
+"Un Mode Solo aux Scénarios interconnectés contre une IA ennemie.\n"
+"Pour les règles spécifiques au Mode Solo\\index{Solo Mode}, voir \\hyperlink{AIrules}{Règles IA}.\n"
```
Comment (qwrtln) [inline translations/game_modes.tex/fr.po:102]: Index entries should be translated. Is this how you want it to be in the Index page?

Comment (qwrtln) [inline translations/game_modes.tex/fr.po:102]:
```suggestion
"Pour les règles spécifiques au Mode Solo\\index{Mode Solo}, voir \\hyperlink{AIrules}{Règles IA}.\n"
```
Or maybe like that?

Comment (Gastonia02) [reply translations/game_modes.tex/fr.po:102]: We were taking the route of keeping indexes the  same as in english as they are not visible

Comment (qwrtln) [reply translations/game_modes.tex/fr.po:102]: What do you mean by that? If you download the printable version from readme, the index is there.

Comment (qwrtln) [reply translations/game_modes.tex/fr.po:102]: You want to have a French version with English index at the end?

Comment (Gastonia02) [reply translations/game_modes.tex/fr.po:102]: Well I didn't know

Comment (qwrtln) [reply translations/game_modes.tex/fr.po:102]: That's OK, that's what I'm here for 😆

**Review by qwrtln — APPROVED** [review]

**Pull request merged**